backends/backends.c

```diff
@@ -997,7 +998,7 @@ void *backends_main(void *ptr) {
                 // usec_t start_ut = now_monotonic_usec();
                 size_t reconnects = 0;
 
-                sock = connect_to_one_of(destination, default_port, &timeout, &reconnects, NULL, 0);
+                sock = connect_to_one_of(destination, &do_not_reconnect, default_port, &timeout, &reconnects, NULL, 0);
```

Comment:
Are you actually using `do_not_reconnect` to not try reconnection? Or am I missing something?

Reply:
Never mind. I see it's in the low lever connect (the different variable name fooled me for a second) :smile:

Reply:
Probably the name is not good, but this is a variable used to identify the address that is invalid and we won't connect again. This is used as a flag, where each one of the 32 positions identify one of the 32 possible address in the list that tried to reconnect. Every time that the address is invalid(return -2 ) I enable the bit and I do not try to reconnect again. Should we rename the variable?